decidim-admin/app/permissions/decidim/admin/permissions.rb

```diff
@@ -66,6 +72,10 @@ def permissions
 
       private
 
+      def user_valuator?
+        user && !user.admin? && Decidim::ParticipatoryProcessUserRole.exists?(user:, role: "valuator")
```

Comment:
If a user has the role of "valuator" for a process and the role of "admin" for another one, this method will return "true".

I don't know what should happen in this situation. But I think it would make sense that if the user has an "admin" role for a participatory process, this user should be allowed to access the global moderations.

Reply:
Indeed, a third argument is required to `Decidim::ParticipatoryProcessUserRole` method.

Reply:
I was looking a bit close here, and i would disagree with your observation . The `user_valuator?` method tries to see is the current user is *not* global admin, yet has *any* valuator role. 
The breakdown of the statement is as follows:
- If there is an user 
- and the user is not a global admin 
- and there is a valuator role for the user

As you can see in this file, at line 33 if the user_valuator becomes true, then the global moderation action would be disallowed. 

Maybe i have missed something in the logic. Could you explain at what point being admin of a process could change the logic ?

Reply:
I agree with @fblupi, you are currently only checking if the user is valuator in any process, but it is possible they are valuator in process A and admin in process B in which case the rule above would not work.

Also note that this code will fail if you do not have `decidim-participatory_processes` installed, as in some system where you are only using assemblies.